docs/requirements/TestRequirements.md

```diff
@@ -130,13 +130,13 @@ Requirements on writing test specifications
 These are some requirements for writing specifications for "Zonemaster":
 
  1. Follow the framework of the IEEE 829-2008.
- 2. The documents must be in [Markdown Syntax]
-    (http://daringfireball.net/projects/markdown/syntax).
+ 2. The documents must be in 
+    [Markdown Syntax](http://daringfireball.net/projects/markdown/syntax).
```

Comment:
Why  [Markdown Syntax] in newline ?

Reply:
There must not be any white space or line break between "]" and "(". There are three options that give the same result on Github:

```
 2. The documents must be in 
     [Markdown Syntax](http://daringfireball.net/projects/markdown/syntax).

 2. The documents must be in [Markdown Syntax
     ](http://daringfireball.net/projects/markdown/syntax)

 2. The documents must be in [Markdown Syntax](
     http://daringfireball.net/projects/markdown/syntax)
```

Reply:
There's also:

```
The documents must be in [Markdown Syntax].

[Markdown Syntax]: http://daringfireball.net/projects/markdown/syntax
```

https://spec.commonmark.org/0.28/#reference-link